src/transformer.ts

```diff
@@ -669,18 +687,35 @@ export default class Transformer {
     return generatedImports;
   }
 
-  resolveSelectImportAndZodSchemaLine(modelName: string) {
+  resolveSelectIncludeImportAndZodSchemaLine(modelName: string) {
     const selectImport = Transformer.isGenerateSelect
       ? `import { ${modelName}SelectObjectSchema } from './objects/${modelName}Select.schema'`
       : '';
+    const includeImport = Transformer.isGenerateInclude
+      ? `import { ${modelName}IncludeObjectSchema } from './objects/${modelName}Include.schema'`
+      : '';
 
     let selectZodSchemaLine = '';
+    let includeZodSchemaLine = '';
     let selectZodSchemaLineLazy = '';
+    let includeZodSchemaLineLazy = '';
     if (Transformer.isGenerateSelect) {
       let zodSelectObjectSchema = `${modelName}SelectObjectSchema.optional()`;
       selectZodSchemaLine = `select: ${zodSelectObjectSchema},`;
       selectZodSchemaLineLazy = `select: z.lazy(() => ${zodSelectObjectSchema}),`;
     }
-    return { selectImport, selectZodSchemaLine, selectZodSchemaLineLazy };
+    if (Transformer.isGenerateInclude) {
+      let zodIncludeObjectSchema = `${modelName}IncludeObjectSchema.optional()`;
```

Comment:
Very minor: `zodIncludeObjectSchema` and `zodSelectObjectSchema` in this function can be `const` instead of `let`

Reply:
Fixed